openassessment/xblock/test/base.py

```diff
@@ -172,3 +201,138 @@ class XBlockHandlerTransactionTestCase(XBlockHandlerTestCaseMixin, TransactionCa
     use `XBlockHandlerTestCase` instead.
     """
     pass
+
+
+class SubmitAssessmentsMixin(object):
+    """
+    A mixin for creating a submission and peer/self assessments so that the user can
+    receive a grade. This is useful for getting into the "waiting for peer assessment" state.
+    """
+    maxDiff = None
+
+    PEERS = ['McNulty', 'Moreland']
+
+    ASSESSMENTS = [
+        {
+            'options_selected': {u'𝓒𝓸𝓷𝓬𝓲𝓼𝓮': u'ﻉซƈﻉɭɭﻉกՇ', u'Form': u'Fair'},
+            'criterion_feedback': {
+                u'𝓒𝓸𝓷𝓬𝓲𝓼𝓮': u'Peer 1: ฝﻉɭɭ ɗѻกﻉ!'
+            },
+            'overall_feedback': u'єאςєɭɭєภՇ ฬ๏гк!',
+        },
+        {
+            'options_selected': {u'𝓒𝓸𝓷𝓬𝓲𝓼𝓮': u'ﻉซƈﻉɭɭﻉกՇ', u'Form': u'Fair'},
+            'criterion_feedback': {
+                u'𝓒𝓸𝓷𝓬𝓲𝓼𝓮': u'Peer 2: ฝﻉɭɭ ɗѻกﻉ!',
+                u'Form': u'Peer 2: ƒαιя נσв'
+            },
+            'overall_feedback': u'Good job!',
+        },
+    ]
+
+    SUBMISSION = (u'ՇﻉรՇ', u'รપ๒๓ٱรรٱѻก')
+
+    STEPS = ['peer', 'self']
+
+    def _create_submission_and_assessments(
```

Comment:
Nit: remove the "private" underscore prefix is you didn't use it for submit_staff_assessment (or set_staff_access). The methods should be consistent (what about set_mock_workflow_info?).